src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1101,11 +1101,18 @@
         <summary>SyntaxToken representing the optional ref or out keyword.</summary>
       </PropertyComment>
     </Field>
-    <Field Name="Expression" Type="ExpressionSyntax">
+    <Field Name="Expression" Type="ExpressionSyntax" Optional="true">
       <PropertyComment>
         <summary>ExpressionSyntax node representing the argument.</summary>
       </PropertyComment>
     </Field>
+    <Field Name="Type" Type="TypeSyntax" Optional="true"/>
+    <Field Name="Identifier" Type="SyntaxToken" Optional="true">
```

Comment:
The outvar.md already lists this as an open issue.

---
In reply to: [63748635](https://github.com/dotnet/roslyn/pull/11393#discussion_r63748635) [](ancestors = 63748635)